bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/OpenUVHandlerFactory.java

```diff
@@ -42,7 +42,8 @@
 @Component(service = ThingHandlerFactory.class, configurationPid = "binding.openuv")
 public class OpenUVHandlerFactory extends BaseThingHandlerFactory {
 
-    private @Nullable LocationProvider locationProvider;
+    @NonNullByDefault({})
+    private LocationProvider locationProvider;
```

Comment:
Please keep the pattern "private @NonNullByDefault({}) LocationProvider locationProvider;"